service/history/configs/config.go

```diff
@@ -356,6 +356,9 @@ type Config struct {
 
 	UseExperimentalHsmScheduler dynamicconfig.BoolPropertyFnWithNamespaceFilter
 	HsmSchedulerTweakables      dynamicconfig.TypedPropertyFnWithNamespaceFilter[schedulerhsm.Tweakables]
+
+	HealthPersistenceLatencyFailure dynamicconfig.FloatPropertyFn
```

Comment:
some concern as the one in the other PR. I feel this is control plane logic.

Reply:
let's sync offline